lib/anagram_cases.rb

```diff
@@ -4,16 +4,16 @@ def test_name
   end
 
   def work_load
-    actual = "Anagram.new(\"#{subject}\").match(#{candidates})"
-    "assert_equal #{expected}, #{actual}"
+    actual = "Anagram.new('#{subject}').match(#{candidates})"
+    "assert_equal #{expected}.sort, #{actual}.sort"
   end
 
   def skipped
     index.zero? ? '# skip' : 'skip'
   end
 
   def show_comment
-    comment.nil? ? '' : "# #{comment}\n\t\t"
+    comment.nil? ? '' : "# #{comment}"
```

Comment:
Try moving the comment outputing into the `work_load`, this should allow you to avoid empty lines.